src/generator/vhdl/mod.rs

```diff
@@ -51,6 +88,12 @@ pub struct VHDLBackEnd {
     config: VHDLConfig,
 }
 
+impl From<VHDLConfig> for VHDLBackEnd {
+    fn from(config: VHDLConfig) -> Self {
+        VHDLBackEnd { config }
+    }
+}
+
 impl GenerateProject for VHDLBackEnd {
     fn generate(&self, project: &Project, path: &Path) -> Result<()> {
```

Comment:
`Path` arguments in functions are more flexible when expressed as `impl AsRef<Path>`. This is also what the std uses for functions that take a path as argument. This allows callers to also for example use `str`. (https://doc.rust-lang.org/stable/std/path/struct.Path.html#implementations)